job/validators.py

```diff
@@ -185,11 +185,13 @@ def clean_fields(self):
     def clean(self):
         """
         Run all models' clean methods
+        Run ElectricTariff clean method in cross-clean
```

Comment:
I think we should stick to the paradigm where clean() calls all the models' self-contained clean functions. Anything going into cross-clean should be named as such, take the existing cross_clean_pv() as an example.

Reply:
Actually maybe it's not important enough to spend time on right now. But, if we're only cleaning ElectricTariff if off_grid_flag is True, shouldn't we be doing the same thing with ElectricUtility?

Reply:
I can try to find a way where clean() doesnt break if electric tariff is missing from inputs and off-grid is true. And we can create a new function to cross-clean electric tariff.

Reply:
Updated per https://github.com/NREL/REopt_API/pull/328/commits/2c3e977834c2671ce76fcfdd32faed8dbe91d7f6

Reply:
Tested to make sure that providing ElectricUtility does not break off_grid scenarios